### ISSUE TITLE
Import Attendance Error

### ISSUE THREAD

**Issue by jawkhan** [smsalah/hr_pyzk#1]
When I click on Import Attendance from the wizard, I get the following error:
ValueError: astimezone() cannot be applied to a naive datetime

The error is in regards to line 144 in pyzk_machine.py

Please advise.

**Comment by smsalah** [comment]
> When I click on Import Attendance from the wizard, I get the following error:
> ValueError: astimezone() cannot be applied to a naive datetime
> 
> The error is in regards to line 144 in pyzk_machine.py
> 
> Please advise.

Please make sure you have defined timezone in the user preferences.
![timezone](https://user-images.githubusercontent.com/3519795/51983658-187f7000-24bb-11e9-90d2-2fe24c3e5c75.png)

**Comment by smsalah** [comment]
> > When I click on Import Attendance from the wizard, I get the following error:
> > ValueError: astimezone() cannot be applied to a naive datetime
> > The error is in regards to line 144 in pyzk_machine.py
> > Please advise.
> 
> Please make sure you have defined timezone in the user preferences.
> ![timezone](https://user-images.githubusercontent.com/3519795/51983658-187f7000-24bb-11e9-90d2-2fe24c3e5c75.png)

I am also updating the module for both version 11 and version 12. you will get it very soon.

**Comment by jawkhan** [comment]
Yep, I updated my timezone to my local timezone, but still get that error. I look forward to the updated module.
thanks.
